AngularLanguageService.Shared/LanguageServer/CompletionProvider.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.LanguageServer.Client;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Utilities;
+using Microsoft.WebTools.Languages.Html.Editor.Completion;
+using Microsoft.WebTools.Languages.Html.Editor.Completion.Def;
+
+namespace AngularLanguageService.LanguageServer
+{
+	[HtmlCompletionProvider("Children", "*")]
+	[ContentType(ContentDefinitions.AngularComponentContentTypeName)]
+	internal sealed class CompletionProvider : IHtmlCompletionListProvider
```

Comment:
Kind of interesting this works in VS2022.. IIRC they were moving to the async completion provider but maybe this is back compatible with that.

Reply:
Last time I checked they decided to just focus on HTML LSP, since that would fix the async issue.

That reminds me that once they've fully migrated to LSP, we'll be able to remove this shim.